requirements.txt

```diff
@@ -35,6 +35,9 @@ google-api-python-client==1.6.4
 Babel==2.5.1
 citeproc-py==0.4.0
 boto3==1.4.7
+#Using own fork of django-waffle until https://github.com/jsocol/django-waffle/pull/266 is merged
+git+https://github.com/pattisdr/django-waffle.git
```

Comment:
I think we can get off a fork now; looks like they fixed compatibility with new versions of Django: https://github.com/jsocol/django-waffle/pull/269